runtime/vara/src/tests.rs

```diff
@@ -18,17 +18,547 @@
 
 use super::*;
 use crate::Runtime;
-use frame_support::{dispatch::GetDispatchInfo, traits::StorageInstance};
+
+use frame_support::dispatch::GetDispatchInfo;
 use frame_system::limits::WeightsPerClass;
 use gear_core::costs::LazyPagesCosts;
 use pallet_gear::{InstructionWeights, MemoryWeights, SyscallWeights};
 use pallet_staking::WeightInfo as _;
-use runtime_common::weights::{
-    check_instructions_weights, check_lazy_pages_costs, check_pages_costs, check_syscall_weights,
-    PagesCosts,
-};
 use sp_runtime::AccountId32;
 
+#[cfg(feature = "dev")]
+use frame_support::traits::StorageInstance;
```

Comment:
pls move everything into module. runtime/vara/tests is general testing module. so all "utils" like `WeightExpectation' should be somewhere else. keep here tests only

Reply:
Done